tests/fdb/test_fdb.py

```diff
@@ -25,13 +25,25 @@
 DEFAULT_FDB_ETHERNET_TYPE = 0x1234
 DUMMY_MAC_PREFIX = "02:11:22:33"
 DUMMY_MAC_COUNT = 10
+DUMMY_MAC_COUNT_SLIM = 3
 FDB_POPULATE_SLEEP_TIMEOUT = 2
 FDB_CLEAN_UP_SLEEP_TIMEOUT = 2
 FDB_WAIT_EXPECTED_PACKET_TIMEOUT = 5
 PKT_TYPES = ["ethernet", "arp_request", "arp_reply", "cleanup"]
 
 logger = logging.getLogger(__name__)
 
+@pytest.fixture(scope="module")
+def get_dummay_mac_count(tbinfo):
+    REQUIRED_TOPO = ["t0-116"]
+    if tbinfo["topo"]["name"] in REQUIRED_TOPO:
+        # To reduce the case running time and avoid mac age issue
```

Comment:
This case has disabled fdb age.
https://github.com/Azure/sonic-mgmt/blob/master/tests/fdb/test_fdb.py#L22

Reply:
Yes, should be not related with fdb age, changed the commit message~